builder/index.html

```diff
@@ -381,10 +381,10 @@ <h5>
                   </div>
 
                   <div class="col">
+                    <ul id="methods" class="list-unstyled" dnd-list="selectedTechnique.method_calls" dnd-drop="dropCallback(item, index, type);" >
```

Comment:
is it working if you drag over the notification ? Shoud the notification should not be the first li element of the ul ?

Reply:
It's working well but it would be more w3c compliant if this div was a <li> element, I will change that!